package.json

```diff
@@ -252,8 +252,8 @@
     "p-limit": "^3.1.0",
     "path-to-regexp": "^6.2.0",
     "proper-lockfile": "^4.1.2",
-    "react": "^18.0.0",
-    "react-dom": "^18.0.0",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
```

Comment:
I think having latest `react@18` with outdated types from `"@types/react-dom": "^17.0.14` quite acceptable.. unless we figure out how to use new types? 
I would keep latest `react` anyway since it should be backward compatible yet for some outdated react apis, e.g. `ReactDOM.render` should be still available.

Reply:
We found that both the lenscloud extension and the Kube resource map extension throw errors after this upgrade without trying to fix them.